snapcraft/tests/test_plugin_copy.py

```diff
@@ -332,6 +332,10 @@ def test_copy_symlinks_that_should_be_followed(self):
                 with open(destination, 'r') as f:
                     self.assertEqual(f.read(), symlink['expected_contents'])
 
+    def test_copy_enable_cross_compilation(self):
+        c = CopyPlugin('copy', self.mock_options, self.project_options)
+        c.enable_cross_compilation()
```

Comment:
This test verifies that this plugin does not stop when the target architecture is different from host architecture. Both plugins do the same on all architectures. The other features already have their own tests.